### PR TITLE
CLIP-1575: Product version upgrade

### PR DESCRIPTION
**Pull request by jjeongatl**
## Pull request description

Product version upgraded for release 1.5.0
Date error corrected in Changelog

## Checklist
- [ ] I have added unit tests
- [ ] I have applied the change to all applicable products
- [ ] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [ ] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/main/charts/bamboo-agent/Changelog.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## 1.3.0
 
-**Release date:** 2022-23-03
+**Release date:** 2022-03-03
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:17]:
```suggestion
**Release date:** 2022-03-24
```

Comment (nghazali) [reply src/main/charts/bamboo-agent/Changelog.md:17]: The release happened in March 23 (more precise is March 24 based on GitHub release history)

File: src/main/charts/bamboo/Changelog.md [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 ## 1.3.0
 
-**Release date:** 2022-23-03
+**Release date:** 2022-03-03
```
Comment (nghazali) [inline src/main/charts/bamboo/Changelog.md:27]:
```suggestion
**Release date:** 2022-03-24
```

File: src/main/charts/confluence/Changelog.md [hunk 3]
```diff
@@ -40,7 +40,7 @@
 
 ## 1.3.0
 
-**Release date:** 2022-23-03
+**Release date:** 2022-03-03
```
Comment (nghazali) [inline src/main/charts/confluence/Changelog.md:43]:
```suggestion
**Release date:** 2022-03-24
```

**Review by nghazali — CHANGES_REQUESTED** [review]

File: src/main/charts/bamboo-agent/Changelog.md (hunk 1 above)
Comment (nghazali) [reply src/main/charts/bamboo-agent/Changelog.md:17]: Please check for other products as well.

Comment (yzha645) [inline src/main/charts/bamboo-agent/Changelog.md:17]: You meant `2022-03-23`?

File: src/main/charts/bamboo-agent/Chart.yaml [hunk 4]
```diff
@@ -26,7 +26,7 @@ annotations:
       - name: Github PR
         url: https://github.com/atlassian/data-center-helm-charts/pull/389
     - kind: changed
-      description: Bamboo Agent updated to 8.2.3 version
+      description: Bamboo Agent updated to 8.2.4 version
       links:
       - name: Github PR
         url: https://github.com/atlassian/data-center-helm-charts/pull/412
```
Comment (yzha645) [inline src/main/charts/bamboo-agent/Chart.yaml:32]:
```suggestion
        url: https://github.com/atlassian/data-center-helm-charts/pull/430
```

File: src/main/charts/bamboo-agent/Changelog.md (hunk 1 above)
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:17]:
```suggestion
**Release date:** 2022-03-24
```

**Review by nghazali — CHANGES_REQUESTED** [review]

**Review by nghazali — APPROVED** [review]

**Review by yzha645 — APPROVED** [review]

**Pull request merged**